JS-112_javascript-arrays/multidimensional-arrays/code-challenge/build-a-quiz-challenge/js/quiz.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * JS-112: JavaScript Arrays - Loop Through Arrays
+ * 
+ * Code challenge: Build A Qiuz Challenge
+ */
+// 1. Create a multidimensional array to hold quiz questions and answers
+const quizItems = [
+  [
+    'The sun is the center of the solar system. (true or false)',
+    'true'
+  ],
+  [
+    'I was once a cucumber. (yes or no)',
+    'yes'
+  ],
+  [
+    'What never asks a question but gets answered all the time?',
+    'cellphone'
+  ],
+  [
+    'What can one catch that is not thrown?',
+    'cold'
+  ],
+  [
+    'Everyone in the world needs it, but they usually give it without taking it. What is it?',
+    'advice'
+  ],
+];
+
+// 2. Store the number of questions answered correctly
+let correct = 0;
```

Comment:
You don't need to store the number of correct answers. You can easily get them by getting the length of the `rightAnswers` array.
```suggestion
```